### PR TITLE
chore(execution-engine): VM execution step negative tests [fixes VM-251]

### PR DESCRIPTION
**Pull request by raftedproc**

### CODE REVIEW

File: air/tests/test_module/negative_tests/execution_step.rs [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+        "#);
+    let unsupported_jvalue = json!({"int": scalar_int, });
+    let mut vm_2 = create_avm(set_variable_call_service(unsupported_jvalue.clone()), vm_2_peer_id);
+    let result = vm_2.call(&script, "", "", <_>::default()).unwrap();
+    let expected_error = CatchableError::FoldIteratesOverNonArray(json!(scalar_int), ".$.int".to_string());
+    assert!(check_error(&result, expected_error));
+}
+
+#[test]
+fn nonstring_in_triplet_for_call() {
```
Comment (mikevoronov) [inline air/tests/test_module/negative_tests/execution_step.rs:139]: Let's add 6 more test cases where all of these put into other 2 positions in a `call` triplet.

File: air/tests/test_module/negative_tests/execution_step.rs [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+        .call(script, "", result.data, client_peer_id, 0, 0, None, call_results_4_call)
+        .unwrap();
+
+    let err_msg = std::rc::Rc::new(f!("\"{err_msg}\""));
+    let expected_error = CatchableError::LocalServiceError(10000, err_msg);
+    assert!(check_error(&result, expected_error));
+}
+
+#[test]
+fn ap_scalar_variable_not_found_error() {
```
Comment (mikevoronov) [inline air/tests/test_module/negative_tests/execution_step.rs:49]: Suggest naming tests in this file following this pattern
`<error_name>_<specific>`, so this one would be `variable_not_found_ap_scalar`, and without `error`.

File: air/tests/test_module/negative_tests/execution_step.rs (hunk 1 above)
Comment (raftedproc) [reply air/tests/test_module/negative_tests/execution_step.rs]: Done

File: air/tests/test_module/negative_tests/execution_step.rs (hunk 2 above)
Comment (raftedproc) [reply air/tests/test_module/negative_tests/execution_step.rs]: Done

**Pull request closed without merging**